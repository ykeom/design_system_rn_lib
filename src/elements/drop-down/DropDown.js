import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class DropDown extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>drop down</Text>
      </View>
    );
  }
}

export default DropDown;
