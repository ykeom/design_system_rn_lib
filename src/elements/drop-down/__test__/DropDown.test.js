import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import DropDown from '../DropDown';

describe('DropDown', () => {
  it('should render', () => {
    const component = shallow(<DropDown />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
