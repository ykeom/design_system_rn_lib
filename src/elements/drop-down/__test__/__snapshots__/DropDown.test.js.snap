// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDown should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    drop down
  </Text>
</View>
`;
